alacritty/src/renderer/glyph.rs

```diff
@@ -0,0 +1,407 @@
+use super::math::*;
+use crate::config::font::{Font, FontDescription};
+use crate::config::ui_config::Delta;
+use crate::config::Config;
+use crate::cursor;
+use alacritty_terminal::ansi::CursorStyle;
+use alacritty_terminal::term::CursorKey;
+use crossfont::{FontDesc, FontKey, Rasterize, Rasterizer, Size, Slant, Style, Weight};
+use fnv::FnvHasher;
+use log::*;
+use std::collections::HashMap;
+use std::hash::BuildHasherDefault;
```

Comment:
I suggest to look into other files in our codebase wrt `'use` order. In general it's more like

```
use std::something*

use somecrate::*

use alacritty_terminal

mod some_module

use some_module
```

Since you chaotic order in like every new file, I'd suggest to fix the rest as well.